### ISSUE TITLE
Pronounced gut microbiota signatures in patients with JAK2V617F-positive essential thrombocythemia

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#245]
Pronounced gut microbiota signatures in patients with JAK2V617F-positive essential thrombocythemia – Christina Schjellerup Eickhardt-Dalbøge – Microbiology Spectrum
https://journals.asm.org/doi/10.1128/spectrum.00662-23

**Comment by RotimiDesire** [comment]
This is ready for review now @SvetlanaUP @Omabekee 

I have implemented all corrections from my last curation and I hope I got it better this time.

**Comment by Scholarpat** [comment]
Hi Svet,

I've made some minor changes to this curation for completeness:

**Study Design**: The study compared healthy controls to essential thrombocythemia (ET) patients, who were then subdivided according to mutation status into JAK2V617F-positive and JAK2V617F-negative. These groups were compared to healthy controls and against each other. Therefore, a case-control design is more appropriate than a cross-sectional design that was initially curated.

**Alpha Diversity:** The Inverse Simpson index was left out in Experiment 1. Although the comparison was not significant, it was conducted. I have marked this section as unchanged.

I included a couple of missing taxa and altered the group names and definitions for clarity.

I've included a report [here](https://github.com/waldronlab/BugSigDB/issues/230#issue-2327741392) about an issue I noticed while curating.

Thank you.
